packages/angular/package.json

```diff
@@ -57,7 +57,7 @@
     "jasmine-spec-reporter": "~7.0.0",
     "karma": "~6.3.9",
     "karma-chrome-launcher": "~3.1.0",
-    "karma-coverage-istanbul-reporter": "~3.0.3",
+    "karma-coverage": "~2.1.0",
```

Comment:
@angular/cli@13 drops the `karma-coverage-istanbul-reporter` in favor of this one.